paris-material/src/main/java/com/airbnb/paris/material/proxies/MaterialButtonProxy.kt

```diff
@@ -0,0 +1,161 @@
+package com.airbnb.paris.material.proxies
+
+import android.content.res.ColorStateList
+import android.graphics.PorterDuff
+import android.graphics.drawable.Drawable
+import android.util.Log
+import android.util.TypedValue
+import androidx.annotation.Px
+import androidx.annotation.StyleRes
+import com.airbnb.paris.annotations.AfterStyle
+import com.airbnb.paris.annotations.Attr
+import com.airbnb.paris.annotations.Styleable
+import com.airbnb.paris.material.R
+import com.airbnb.paris.material.R2
+import com.airbnb.paris.material.utils.ViewUtils
+import com.airbnb.paris.proxies.BaseProxy
+import com.airbnb.paris.styles.Style
+import com.google.android.material.button.MaterialButton
+
+@Styleable("Paris_MaterialButton")
+class MaterialButtonProxy(view: MaterialButton) : BaseProxy<MaterialButtonProxy, MaterialButton>(view) {
+
+    private var mColors: ColorStateList? = null
+    private val defaultTextAppearance:Int = 16974317 //todo this might not be the default for every android version or phone
+    private var mTextAppearance = defaultTextAppearance
+
+    companion object{
+
+        @com.airbnb.paris.annotations.Style
+        val RED_STYLE = R.style.Widget_MaterialComponents_Button_OutlinedButton
+    }
+
+    @Attr(R2.styleable.Paris_MaterialButton_android_textColor)
+    fun setTextColor(colors: ColorStateList?) {
```

Comment:
`MaterialButton` extends from `TextView` so all the attributes supported by `TextViewProxy` and `ViewProxy` should already work. They don't need to be added again here.